src/analysis/normed/ring/seminorm.lean

```diff
@@ -164,6 +165,26 @@ end
 
 end ring
 
+lemma is_nonarchimedean.map_nat_cast_le_one {R F : Type*} [non_assoc_ring R]
```

Comment:
Is `map` really a good thing to have in this lemma name? `map` usually means something else (i.e. `functor.map`)

Reply:
The previous lemma is using `seminorm` to denote `f`, not `map`, and you should probably stick with the already established naming conventions established.